cocotb/utils.py

```diff
@@ -53,28 +53,35 @@ def _get_simulator_precision():
 def get_python_integer_types():
     warnings.warn(
         "This is an internal cocotb function, use six.integer_types instead",
-        DeprecationWarning)
+        DeprecationWarning, stacklevel=2)
     return (int,)
 
 
 # Simulator helper functions
-def get_sim_time(units=None):
+def get_sim_time(units: str = "step") -> int:
     """Retrieves the simulation time from the simulator.
 
     Args:
-        units (str or None, optional): String specifying the units of the result
+        units: String specifying the units of the result
             (one of ``None``, ``'fs'``, ``'ps'``, ``'ns'``, ``'us'``, ``'ms'``, ``'sec'``).
             ``None`` will return the raw simulation time.
```

Comment:
```suggestion
            (one of ``'step'``, ``'fs'``, ``'ps'``, ``'ns'``, ``'us'``, ``'ms'``, ``'sec'``).
            ``'step'`` will return the raw simulation time.
 
            .. deprecated:: 1.6.0
                Using None as the *units* argument is deprecated, use ``'step'`` instead.
```

Reply:
`get_sim_steps` never got this section. Guess I should add it there as well.